RestApi/Sepes.RestApi/Model/Study.cs

```diff
@@ -1,12 +1,19 @@
+using System.Collections.Generic;
+
 namespace Sepes.RestApi.Model
 {
 
     public class Study
     {
         public string studyName { get; set; }
         public int studyId { get; set; }
+        public List<Pod> pods { get; set; }
+        public List<string> owners { get; set; }
+        public List<string> users { get; set; }
+        public bool archived { get; set; }
```

Comment:
I think we need to have a talk around if modules should be mutable(set and get) or immutable(get only).
The input to controllers must be mutable as ASP need to be able to create them.
But i feel that internal models should be immutable. It makes changes explicit as they generate new model and not mutate the model in place.